bls/bignum_hol256.go

```diff
@@ -14,7 +14,6 @@ var _modulus u256.Int
 type Fr u256.Int
 
 func init() {
-	SetFr((*Fr)(&_modulus), "52435875175126190479447740508185965837690552500527637822603658699938581184513")
```

Comment:
I think we still need this line because bignum modulus has a different type from `MODULUS`.

Reply:
yup missed that. changed & made sure it now builds for each -tag variant.